docs/csharp/roslyn-sdk/syntax-visualizer.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Explore code with the syntax visualizer
+description: The syntax visualizer provides a visual tool to explore the models the .NET Compiler Platform SDK generates for code.
+author: billwagner
+ms.author: wiwagn
+ms.date: 03/07/2018
+ms.topic: conceptual
+ms.prod: .net
+ms.devlang: devlang-csharp
+ms.custom: mvc
+---
+# Explore code with the syntax visualizer
+
+This article provides an overview of the Syntax Visualizer tool that ships as part of the .NET Compiler Platform ("Roslyn") SDK. The Syntax Visualizer is a tool window that helps you inspect and explore syntax trees. It's an essential tool to understand the models for code you want to analyze. It's also a debugging aid when you develop your own applications using the .NET Compiler Platform (“Roslyn”) SDK. Open this tool as you create your first analyzers. The visualizer helps you understand the models used by the APIs.
+
+[!INCLUDE[interactive-note](~/includes/roslyn-installation.md)]
+
+Familiarize yourself with the concepts used in the .NET Compiler Platform SDK by reading the [overview](compiler-api-model.md) article. It provides an introduction to syntax trees, nodes, tokens, and trivia.
+
+## Syntax Visualizer
+
+The **Syntax Visualizer** enables inspection of the syntax tree for the C# or VB code file in the current active editor window inside the Visual Studio IDE. The visualizer can be launched by clicking on **View** –> **Other Windows** –> **Syntax Visualizer**.  You can also use the **Quick Launch** toolbar in the upper right corner. Type "syntax" and the command to open the **Syntax Visualizer** should appear.
```

Comment:
We are [supposed to use](https://worldready.cloudapp.net/Styleguide/Read?id=2700&topicid=26472) the greater than sign (>) for successive menu choices (e.g. View > Other Windows > etc.).